qa-tests-backend/src/test/groovy/DeclarativeConfigTest.groovy

```diff
@@ -196,10 +196,6 @@ oidc:
 name: ${AUTH_PROVIDER_KEY}
 minimumRole: "None"
 uiEndpoint: localhost:8000
-groups:
-- key: "email"
-  value: "someone@example.com"
-  role: "Admin"
```

Comment:
why removing this?

Reply:
It's actually not required, so removed it to simplify it.